Fix LMS colors

cristian-ungureanu opened the pull request:
### Summary
Seems like Lifter LMS has hardcoded colors in its code, making our dark pallet not working as it is supposed to work.

### Will affect visual aspect of the product
YES

### Test instructions
- Install Lifter LMS and import their courses
- Import a starter site, add the pallet switcher component
- Check if colors are looking ok for both Memberships and Courses
- Check the dashboard page of Lifter
- Check achievements and certificates

<!-- Issues that this pull request closes. -->
Closes Codeinwp/neve-pro-addon#1810.
<!-- Should look like this: `Closes #1, #2, #3.` . -->

cristian-ungureanu commented:
@abaicus I'm not really sure about the way I wrote the code here, I've added the same selectors that they have to overwrite the CSS but maybe you can have a look and let me know if we can better group them.

abaicus reviewed (approved):
I think this looks good 👍🏻

irinelenache commented:
@cristian-ungureanu Tested this with **Web agency** imported and found a few issues:

- [ ] The buttons on the **Dashboard** page don't look ok in dark mode https://vertis.d.pr/Zfz8pf
- [ ] The **progress percent** and the **update button** on the **My grades** page don't look ok in dark mode https://vertis.d.pr/Wu5wsg
- [ ] The **progress bar of the course** still uses the original color https://vertis.d.pr/WpyW0v https://vertis.d.pr/wvNONy
- [ ] The colors on the **quiz questions** are not visible in dark mode https://vertis.d.pr/EJh4rT
- [ ] The colors on the **certificate** are not visible in dark mode https://vertis.d.pr/vn2cS1

cristian-ungureanu commented:
@irinelenache I think when you've tested, you had black for the text color on the secondary button.

1. This is how it looks when the button colors are not linked with the global colors https://vertis.d.pr/Nyf31T ( if you toggle the pallet, they won't change ). This is how they look when you use global colors: https://vertis.d.pr/QWqHR3
2. Same for the update button ( https://vertis.d.pr/RFGJXZ ), fixed the progress percent 
3. Fixed, now controlled from Customizer -> Layout -> LifterLMS -> Colors
4. Fixed

irinelenache commented:
@cristian-ungureanu Tested again and the reported issues are fixed, thank you 🚀

cristian-ungureanu commented:
Thank you for testing, Irinel! 🚀
